client/my-sites/domains/domain-management/dns/txt-record.jsx

```diff
@@ -2,11 +2,14 @@ import classnames from 'classnames';
 import { localize } from 'i18n-calypso';
 import PropTypes from 'prop-types';
 import { Component } from 'react';
+import ExternalLink from 'calypso/components/external-link';
 import FormFieldset from 'calypso/components/forms/form-fieldset';
 import FormInputValidation from 'calypso/components/forms/form-input-validation';
 import FormLabel from 'calypso/components/forms/form-label';
 import FormTextInputWithAffixes from 'calypso/components/forms/form-text-input-with-affixes';
 import FormTextarea from 'calypso/components/forms/form-textarea';
+import { localizeUrl } from 'calypso/lib/i18n-utils';
```

Comment:
I think you don't need to call `localizeUrl` at all because it is called in the client/lib/url/support.js file where the `DNS_TXT_RECORD_CHAR_LIMIT ` constant is defined

Reply:
Thanks, fixed!